tdp_vars_overrides/hue/hue.yml

```diff
@@ -7,3 +7,6 @@ ldapauth:
   users:
     user_filter: objectclass=posixAccount
     user_name_attr: uid
+
+# Hue release for CentOS7 (Python2.7), Rocky8 (Python3.6) or later
+hue_dist_file: "{% if (ansible_distribution_major_version | int) > 7 %}{{ hue_release }}-python36.tar.gz{% else %}{{ hue_release }}.tar.gz{% endif %}"
```

Comment:
I think this should be in repo `tdp-collection-extras` no ?

Reply:
In ```tdp-collection-extras``` repo, we would need to modify this in ```tdp_vars_defaults```.